<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverwait</class>
 <widget class="QWidget" name="serverwait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(176, 224, 230);</string>
  </property>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>102</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name2">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>172</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name3">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>242</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="waiting">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>waiting...</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>106</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>176</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>246</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name4">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>102</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name5">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>172</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name6">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>242</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>106</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>176</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>246</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
